### PR TITLE
17 student card view

### PR DESCRIPTION
**Pull request by andrew-wagner89**
To test follow the link on the landing page to see a sample student card
This assumes you are using the mock data DAO's

### CODE REVIEW

**Comment by andrew-wagner89** [comment]
just changed the code so it works with the database DAO and not the mock data, use the studentDao branch on the server now

File: src/views/Student/Student.js [hunk 1]
```diff
@@ -0,0 +1,325 @@
...
+        {
+          Utils.cardKeys.filter(key => key in Utils.studentTranslations).sort(Utils.sortStudentStats).map((key, i) => {
+            return <Col xs={12} md={6}> {this.getFormItem(key, student[key])}</Col>
+          })
+        }
+      </div>
+    )
+  }
+
+  validateScore(key){
```
Comment (jamesvclements) [inline src/views/Student/Student.js]: I think this function belongs in `Utils.js` incase we want to use it elsewhere

Comment (andrew-wagner89) [reply src/views/Student/Student.js]: Yeah probably a good idea, I'll move it gimme a sec

File: src/views/Student/Student.js [hunk 2]
```diff
@@ -0,0 +1,325 @@
...
+    let val = this.state.student[key]
+    let retVal = ''
```
Comment (jamesvclements) [inline src/views/Student/Student.js]: Is using `retVal` necessary? I think we could just return 'error' or 'success' immediately in any of the cases

File: src/views/Student/Student.js [hunk 3]
```diff
@@ -0,0 +1,325 @@
...
+    tempStudent[key] = event.target.value
+    this.setState({student : tempStudent })
+  }
+
+  getFormItem(key, val){
+    // first switch non test scores that expect an empty string
+    if (typeof val === 'undefined' || !val){
+      val = 0
+    }
+    switch (key) {
```
Comment (jamesvclements) [inline src/views/Student/Student.js:153]: Needs a default case

File: src/utils/Utils.js [hunk 4]
```diff
@@ -64,10 +64,12 @@ export const studentTranslations = {
...
   behaviorObservation: 'Behavior Observation',
   dial4: 'Dial 4',
-  age: 'Age'
+  age: 'Age',
+  comments: 'Comments'
 }
 
 export function forHumanAttr(key, val) {
+  //first switch non test scores that expect an empty string
   switch (key) {
```
Comment (jamesvclements) [inline src/utils/Utils.js:73]: needs default case

**Comment by andrew-wagner89** [comment]
@jamesvclements I fixed all of your suggestions but I think I might have blown away some of your refactors, my b

**Comment by jamesvclements** [comment]
Good stuff. I figured out how to wrap the panel around the whole info component so that it matches the edit component but I had to use some hacky CSS to remove the list-group-item borders. In the future I think it would be nice to make the info component match the edit component more (e.g. the headers for the field names are bold and centered above the values) but this is great for now.

**Comment by jamesvclements** [comment]
:shipit:

**Pull request merged**